doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -218,8 +218,8 @@ nRF IEEE 802.15.4 radio driver
 Wi-Fi
 -----
 
-* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board using :ref:`Shield <ug_nrf7002eb_nrf54h20dk_gs>`.
-* Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board using :ref:`Shield <ug_nrf7002eb_nrf54l15pdk_gs>`.
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`.
+* Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`.
```

Comment:
@b-gent : Can we keep the change log entry in this PR because removing the PR would fail the doc build.

Reply:
You can get around the build breaking without the changelog edits by adding these target tags to the `nrf7002eb_dev_guide.rst` if keeping these changes isn't an option.

Reply:
I think we can combine the entries to avoid repetition.

"* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` and:ref:`zephyr:nrf54l15pdk_nrf54l15` boards with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`."

Reply:
> You can get around the build breaking without the changelog edits by adding these target tags to the `nrf7002eb_dev_guide.rst` if keeping these changes isn't an option.

Yeah, I had thought of this. But, since these pages were recently added, thought of getting rid of these labels. But since you also think we could go with this option, I have added these labels to the nrf7002eb_dev_guide.rst. This also avoids editing some of the other pages as well.

Reply:
> I think we can combine the entries to avoid repetition.
> 
> "* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15` boards with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`."

I will add a suggestion to PR https://github.com/nrfconnect/sdk-nrf/pull/15961, since this is now removed from this PR.